backend/matchmaker_service.py

```diff
@@ -193,11 +193,7 @@ def _matchmaker_function(self, sleep_seconds: int = 60, is_test: bool = False):
                     if self._check_start_time(schedule=schedule, now_dt_with_tz=datetime.datetime.now().astimezone(), 
                         is_test=is_test) and now_utc != game_dict.get("last_checked_date"): # TODO: Do these checks in query
                         if game_dict["count_players"] >= self._min_players:
-                            options = GameOptions(game_schedule=schedule, game_wait_sleep_interval_sec=1 if is_test else 30,
-                            single_tribe_council_time_sec=1 if is_test else 300, # is there a way to get the default values from GameOptions?
-                            single_team_council_time_sec=1 if is_test else 300,
-                            final_tribal_council_time_sec=1 if is_test else 300,
-                            multi_tribe_council_time_sec=1 if is_test else 300)
+                            options = GameOptions(game_schedule=schedule, game_wait_sleep_interval_sec=1 if is_test else 30)

```

Comment:
Have you tried testing this with the non-mocked unit tests locally? IIRC the tests would take a very long time to complete unless all the wait times were specified to 1 sec

Reply:
@dzx11 yea there are two modes of operation in the game event loop now [1], synchronous, where it waits for scheduled event times, and asynchronous, where there is no wait (for testing). because of this change we no longer need to set these time intervals.

[1] https://github.com/un7c0rn/stopthevirus/blob/4ab01aae2a7934e6a2ae4e2aaa7d8b54b3784e0a/backend/game.py#L190

Reply:
MM tests do pass.